app/controllers/RegistrationSentController.scala

```diff
@@ -33,12 +36,12 @@ class RegistrationSentController @Inject()(
     requireData: DataRequiredAction,
     view: RegistrationSentView,
     val controllerComponents: MessagesControllerComponents
-  )(implicit appConfig: FrontendAppConfig) extends LocalBaseController {
+  )(implicit appConfig: FrontendAppConfig) extends ImplicitDateFormatter with  LocalBaseController {
 
   def onPageLoad: Action[AnyContent] = (identify andThen getData andThen requireData).async { implicit request =>
     request.userAnswers.get(AcknowledgementReferencePage) match {
       case Some(acknowledgementReference) =>  userAnswerRepository.delete(request.userAnswers).map { _ =>
-        Ok(view(acknowledgementReference))
+        Ok(view(dayToString(LocalDate.now().plusDays(28)),acknowledgementReference))
```

Comment:
please inject time machine object to get local date now

Reply:
modified